kubemarine/__main__.py

```diff
@@ -155,6 +152,11 @@ def import_procedure(name):
     module_name = 'kubemarine.procedures.%s' % name
     return __import__(module_name, fromlist=['object'])
 
+def version():
+
+    from kubemarine.core import utils
+    with open((utils.get_resource_absolute_path("version", script_relative=True), 'version')[0], 'r') as f:
```

Comment:
`(utils.get_resource_absolute_path("version", script_relative=True), 'version')[0]`
Funny obfuscation of the code :)